src/plugins/intel_gpu/src/graph/graph_optimizer/modify_fused_ops.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "pass_manager.h"
+#include "program_node.h"
+
+#ifdef ENABLE_ONEDNN_FOR_GPU
+#include "gemm_inst.h"
+#include "reshape_inst.h"
+#include "softmax_inst.h"
+#include <impls/onednn/utils.hpp>
+#endif
+
+using namespace cldnn;
+
+void modify_fused_ops::run(program& p) {
+#ifdef ENABLE_ONEDNN_FOR_GPU
```

Comment:
why do we have this `ifdef`? I think we need to remove it because it is necessary for cldnn as well.